base/doc/usrguide.tex

```diff
@@ -1312,6 +1312,21 @@ \section{Expandable floating point (and other) calculations}
 the values \cs{topskip} (\dimeval{\topskip}) and \cs{baselineskip}
 (\dimeval{\baselineskip}) in the current document.
 
+\section{Expandable \cs{input} equivalent}
+
+\begin{decl}
+  |\expandableinput| \arg{filename} \\
+\end{decl}
+The \LaTeX{} definition of \cs{input} cannot be used in places where \TeX{} is
+performing expansion: the classic example is at the start of a tabular cell.
+There are a number of reasons for this: the key ones are that \cs{input}
+records which files are read and provides pre- and post-file hooks.
+
+To support the need to carry out file input in expansion contexts, the command
+\cs{expandableinput} is available: this skips recording the file name and any
+file hooks, but otherwise behaves like \cs{input}. In particular, it still uses
```

Comment:
Does this wording mean/imply that "file hooks are recorded". 
If so, what does this mean? 
If not, then this needs rewording.